cgitrc

```diff
@@ -36,4 +36,4 @@ mimetype.svg=image/svg+xml
 
 virtual-root=/cgit
 
-scan-path=/var/git
+include=/etc/cgitrc.repos
```

Comment:
should probably add an example cgitrc.repos to this repo (maybe even one that just includes the scan-path line that was removed from the example cgitrc) and modify the example compose files in this repo to mount it into /etc/cgitrc.repos. I think it is important to keep these containers able to operate separably.